aten/src/ATen/native/native_functions.yaml

```diff
@@ -2833,24 +2833,23 @@
 # log_softmax allows positional dtype, unlike most operators, because kwonly is BC-breaking when loading jit models.
 - func: log_softmax.int(Tensor self, int dim, ScalarType? dtype=None) -> Tensor
   variants: function, method
-
-- func: log_softmax.int_out(Tensor self, int dim, ScalarType? dtype=None, *, Tensor(a!) out) -> Tensor(a!)
-  variants: function
-  dispatch:
-    CompositeExplicitAutograd: log_softmax_out
-
-- func: log_softmax.Dimname(Tensor self, Dimname dim, *, ScalarType? dtype=None) -> Tensor
-  variants: function, method
+  structured_delegate: log_softmax.int_out
 
 - func: _log_softmax(Tensor self, int dim, bool half_to_float) -> Tensor
-  structured_delegate: _log_softmax.out
+  variants: function
 
 - func: _log_softmax.out(Tensor self, int dim, bool half_to_float, *, Tensor(a!) out) -> Tensor(a!)
+  variants: function
+
+- func: log_softmax.int_out(Tensor self, int dim, ScalarType? dtype=None, *, Tensor(a!) out) -> Tensor(a!)
   structured: True
   dispatch:
     CPU: log_softmax_cpu_out
     CUDA: log_softmax_cuda_out
 
+- func: log_softmax.Dimname(Tensor self, Dimname dim, *, ScalarType? dtype=None) -> Tensor
```

Comment:
nit the diff is a bit hard to read here since things were moved around. This one is not changing right?

Reply:
Correct.